src/NLog.Mongo/MongoTarget.cs

```diff
@@ -152,8 +190,8 @@ protected override void Write(IList<AsyncLogEventInfo> logEvents)
                 if (ex.MustBeRethrownImmediately())
                     throw;
 
-                foreach (var ev in logEvents)
-                    ev.Continuation(ex);
+                for (int i = 0; i < logEvents.Count; ++i)
```

Comment:
Curious, why the switch to for loop instead of foreach?

Reply:
The foreach allocates a boxed enumerator because of IList